deploy-board/deploy_board/static/js/components/clusterconfigcomponents.js

```diff
@@ -83,24 +83,21 @@ Vue.component('baseimage-select', {
             warningText: '',
         }
     },
-    props: ['imageNames', 'baseImages', 'selectedImageName', 'selectedBaseImage', 'pinImage', 'pinImageEnabled'],
+    props: ['imageNames', 'baseImages', 'selectedImageName', 'selectedBaseImage', 'pinImage', 'pinImageEnabled', 'accountOwnerId'],
     methods: {
         helpClick: function () {
             this.$emit('help-clicked')
         },
-        tryShowWarning: function (baseImageId, pinImage) {
+        tryShowWarning: function (baseImageId, pinImage, accountOwnerId) {
             if (!pinImage) {
                 this.showWarning = false;
             } else {
-                const ONE_DAY = 1000 * 60 * 60 * 24;
-                let baseImage = this.baseImages.find(i => i.value == baseImageId);
-                let age = Math.round((Date.now() - new Date(baseImage.publishDate)) / ONE_DAY);
-                if (age > 180) {
-                    this.warningText = `The base image configured is over 180 days old (${age} days). Please consider update or opt-in Auto Update (Unpin image).`;
-                    this.showWarning = true;
-                } else {
-                    this.showWarning = false;
-                }
+                const baseImage = this.baseImages.find(i => i.value === baseImageId);
+                const warnings = this.createWarningMessages(baseImage, accountOwnerId);
```

Comment:
we might need to have a common var_name for this `account_number` or `acccount_owner_id` across teletraan and rodimus repos.

Reply:
Rodimus already uses `ownerId` naming, so I used it here, but I think that `account_number` looks more naturally for AWS users, so maybe we should rename all `ownerId` to `account_number`, but this requires some amount of work, so let's move it out of the scope of current project.

I will create a task for it.